tfe/resource_tfe_workspace.go

```diff
@@ -299,6 +326,10 @@ func resourceTFEWorkspaceUpdate(d *schema.ResourceData, meta interface{}) error
 			options.WorkingDirectory = tfe.String(workingDir.(string))
 		}
 
+		if tp, ok := d.GetOk("trigger_prefixes"); ok {
+			options.TriggerPrefixes = append([]string(nil), (tp.([]string))...)
```

Comment:
Whoeps 😉

Reply:
😬 Thanks for catching & fixing that. I added test coverage for updating trigger prefixes.

Reply:
> I added test coverage for updating trigger prefixes.

Disregard that. My commit was not necessary, you'd already made the changes needed. Removed.